pkg/destroy/aws/aws.go

```diff
@@ -265,6 +265,30 @@ func (o *ClusterUninstaller) RunWithContext(ctx context.Context) ([]string, erro
 	return nil, nil
 }
 
+// findUntaggableResources returns the resources for the cluster that cannot be tagged. Any resource that contains
+// a shared tag will be ignored.
+//
+//	deleted - the resources that have already been deleted. Any resources specified in this set will be ignored.
+func (o *ClusterUninstaller) findUntaggableResources(ctx context.Context, iamClient *iam.IAM, deleted sets.String) (sets.String, error) {
+	resources := sets.NewString()
```

Comment:
You might have complaints from the linter about using the deprecated set functionality. If that happens, we'll have to add a `//nolint: <linter name>` here and fix it later.